src/storage/store.js

```diff
@@ -63,7 +63,7 @@ export const SCHEMA = {
       type: "object",
       additionalProperties: false,
       properties: {
-        displayName: { type: "string", pattern: "^[A-Za-z0-9_~ -]{3,32}$" },
+        displayName: { type: "string", pattern: "^(?:[A-Z]|[a-z]|[0-9]|[_~\\-]|\\s){3,32}$" },
         avatarId: { type: "string" },
         pronouns: { type: "string", pattern: "^[a-zA-Z///a-zA-Z]{2,32}$" },
```

Comment:
This seems to still make Chrome complain. After some testing this seems not to raise any complaints in both Chrome & Firefox:
````
displayName: { type: "string", pattern: "^[A-Za-z0-9_~\\s\\-]{3,32}$" },
avatarId: { type: "string" },
pronouns: { type: "string", pattern: "^([a-zA-Z]{1,32}\\/){0,4}[a-zA-Z]{1,32}$" },
```
This pronouns regex is a bit more strict and enforces the A/B form and you can have up to 5 pronouns that I think it's probably enough for what I've seen?

Reply:
If we prefer to keep the previous regex format, this would work:
```
^[a-zA-Z\\/a-zA-Z]{2,32}$
```

Reply:
Interesting! Thanks for looking into alternatives. I'll update the pronouns with your suggestion - its currently had a few incarnations (enforcing one '/', and leaving it out as an option) but I like your suggestion that allows for up to 5 pronouns. 🚀